.github/workflows/master.yml

```diff
@@ -60,23 +62,34 @@ jobs:
         uses: ./.github/actions/pip_installation
       - name: Serial C tests
         run: |
-          python -m pytest -rx -m "not parallel and c" --ignore=printers --ignore=symbolic --ignore=ndarrays
+          python -m pytest -n auto -rx -m "not (parallel or xdist_incompatible) and c" --ignore=printers --ignore=symbolic --ignore=ndarrays
+          pyccel-clean
+          python -m pytest -rx -m "xdist_incompatible and not parallel and c" --ignore=printers --ignore=symbolic --ignore=ndarrays
+          pyccel-clean
```

Comment:
Do we need to explicitly call `pyccel-clean` if we don't use `pytest-xdist`? I've understood that we don't need to.

Reply:
Fixed in fd8c81af